### ISSUE TITLE
choice of IQ modulator

### ISSUE THREAD

**Issue by hartytp** [sinara-hw/MixMod#9]
@dhslichter IIRC you guys are using the [ADL5375](https://www.analog.com/media/en/technical-documentation/data-sheets/ADL5375.pdf) in some of your setups. What was your criteria for selecting it? How much did you consider other modulators?

Looking for now at the usual ADI/HMC/LTC monster, these seem to be the options:

![image](https://user-images.githubusercontent.com/21218399/65945485-f2381880-e42b-11e9-8a93-4ed4229e2d9c.png)

The [LTC5586](https://www.analog.com/media/en/technical-documentation/data-sheets/5589f.pdf) looks like potentially a really nice choice. The LO/sideband suppression is good (>40dB), so is the noise floor, even if they're not quite as nice as the ADL5375. The thing I like about the LTC5586 is that the power dissipation is way lower than the ADI chips, which is helpful for a design I'm thinking about where I'll be a little squeezed for power. But, can you see something I'm missing that could make it a bad choice?

**Comment by hartytp** [comment]
Sorry, missed the obvious, which is that the LTC5586 has way lower basband (IQ) BW, so isn't suitable for the application I have in mind. Still, would be interested to hear your experiences with these chips and any advice you have on them

**Comment by dhslichter** [comment]
@hartytp this chip choice was made ~5 years ago, and I haven't looked into alternatives since then.  However, it was a good selection because of broad IF bandwidth (although this is not as necessary for some of our stuff), low noise, and low I/Q offsets (i.e. carrier/spurious sideband feedthrough is low without compensation required).  Also nice is the I/Q inputs are differential around 500 mV which makes interfacing with DAC outputs more straightforward than the LTC5586.  

We also found that the I/Q inputs mapped pretty linearly onto the I/Q components of the output -- a lot of modulators are not very linear in this regard, so you end up with phase chirps, amplitude compression, etc, unless you predistort.  This is characterized by the EVM on datasheets.  Obviously there will always be a region in which things start to compress, but we found that we could go up to 1 V differential on the I or Q channels with less than 2% amplitude compression relative to a linear fit at low I/Q voltage (it stays within about +/- 0.5% up to 0.8 V on I/Q).  We extracted a temp coefficient for amplitude of -0.0036 dB/C and for phase of 0.24 degrees/C.  Based on all that, we liked the general passive performance and went with it, figuring that predistortion can be used to tweak up more finely as needed.
